docs/appendices/release-notes/5.6.2.rst

```diff
@@ -93,3 +93,9 @@ Fixes
 - Fixed a race condition that could lead to ``ShardCollectContext already
   added`` errors when making a query after a table had been idle without any
   accesses for a while.
+
+- Fixed an issue that caused resolving an unqualified relation to ignore the
+  search order specified by the :ref:`search_path <conf-session-search-path>`
+  and instead of resolving to a view in a schema with higher precedence in the
+  ``search_path``, to resolve to a table in a schema with lower precedence in
+  in the ``search_path``.
```

Comment:
```suggestion
- Fixed an issue when resolving relations. When resolving an unqualified name
  (no explicit schema), it first exhausted the search path looking for tables
  before moving on to views. Now it will correctly look for both table and view
  in each element of the search path before moving onto the next.

  For example, with a search path set to `a, b`, a query on `tbl` will now look for:

  - table `a.tbl`
  - view `a.tbl`
  - table `b.tbl`
  - view `b.tbl`

  Instead of

  - table `a.tbl`
  - table `b.tbl`
  - view `a.tbl`
  - view `b.tbl`
```